File: src/main/java/toast/pine/Extras.java
package toast.pine;

import org.bukkit.entity.LivingEntity;
import org.bukkit.entity.Player;

public class Extras {

    public static double calculateDistance(LivingEntity entity, Player player) {
        double x1 = entity.getX();
        double y1 = entity.getY();
        double z1 = entity.getZ();

        double x2 = player.getX();
        double y2 = player.getY();
        double z2 = player.getZ();

        return Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2) + Math.pow(z2 - z1, 2));
    }

}

File: src/main/java/pine/toast/Extras.kt
package pine.toast

import org.bukkit.entity.LivingEntity
import org.bukkit.entity.Player
import kotlin.math.pow
import kotlin.math.sqrt

object Extras {
    fun calculateDistance(entity: LivingEntity, player: Player): Double {
        val x1 = entity.x
        val y1 = entity.y
        val z1 = entity.z
        val x2 = player.x
        val y2 = player.y
        val z2 = player.z
        return sqrt((x2 - x1).pow(2.0) + (y2 - y1).pow(2.0) + (z2 - z1).pow(2.0))
    }
}
